### PR TITLE
Created docker file for web

### PR DESCRIPTION
**Pull request by nkotibiot**

### CODE REVIEW

**Review by naveenlakshmi — APPROVED** [review]

File: udmd/web/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM nginx:1.21.5-alpine
+
+ENV BUILD_DIR build
```
Comment (mspratt-biot) [inline udmd/web/Dockerfile:3]: This should change to `ENV BUILD_DIR dist/udmd-web`

File: udmd/web/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+RUN apk --no-cache add curl
+
+# Copy the artifacts from the webs build into nginx to be served.
+COPY $BUILD_DIR $WEB_DIR
+
+# Copy the custom nginx template file into the nginx folder
+# to allow envsubst to pick it up and generate a nginx.conf file
+# with the replaced docker runtime env variables.
+COPY prod/nginx.conf /etc/nginx/templates/nginx.conf.template
```
Comment (mspratt-biot) [inline udmd/web/Dockerfile]: This should change to `COPY prod/nginx.conf /etc/nginx`

File: udmd/web/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+# Copy the docker entrypoint file to the root.
+COPY prod/docker-entrypoint.sh /
```
Comment (mspratt-biot) [inline udmd/web/Dockerfile]: Don't need this line.

File: udmd/web/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+# Launch nginx.
+CMD ["nginx", "-g", "daemon off;"]
```
Comment (mspratt-biot) [inline udmd/web/Dockerfile:20]: Don't need this line.

File: udmd/web/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,24 @@
+FROM nginx:1.21.5-alpine
+
+ENV BUILD_DIR build
+ENV WEB_DIR /usr/share/nginx/html 
+ENV HOME_DIR /usr/share/nginx
+
+WORKDIR $WEB_DIR
+
+RUN apk --no-cache add curl
```
Comment (mspratt-biot) [inline udmd/web/Dockerfile:9]: This can be removed also, unless you want to be able to health check the docker image locally.

**Review by mspratt-biot — APPROVED** [review]

**Pull request merged**